package.json

```diff
@@ -12,9 +12,9 @@
     "url": "git@github.com:adobe/reactor-extension-core.git"
   },
   "scripts": {
-    "build": "rm -rf dist && webpack --mode=production",
-    "build:watch": "webpack --mode=development --watch=true",
-    "sandbox": "webpack --env sandbox --mode=development --watch=true",
+    "build": "rm -rf dist && webpack --mode production",
+    "build:watch": "webpack --mode development --watch",
+    "sandbox": "concurrently -k -n webpack,sandbox -c \"bgBlue,bgMagenta\" \"npm run build:watch\" \"reactor-sandbox\"",
```

Comment:
Latest webpack doesn't allow passing custom parameters. This blocks us launching the sandbox after the webpack compiles the files. On other extensions I went around this issue by launching sandbox and webpack in parallel. I am open to other ideas if this approach causes issues.

Reply:
works for now